### DISCUSSION TITLE
Error: connect ECONNREFUSED 127.0.0.1:33205

### DISCUSSION THREAD

**Discussion by hyper-dot** [vercel/next.js#51448] [Help]
### Summary

I am getting this error on fresh install. I didnt use typescript, eslint , src dir. 
I have project dir. In which backend and frontend dirs are located. I installed next in frontend dir. On starting the server I am getting this error.
```

Uncaught Error: connect ECONNREFUSED 127.0.0.1:33205
client.js:1:4803

```
I have no clue why it is accessing port 33205.

**Comment by Greek** [comment]
Bump
